### PR TITLE
Feat/229/improve view of applicants postullated in projects

### PR DESCRIPTION
**Pull request by AxelUZ**
# What does this PR do?
Improves the way applicants are displayed in projects and extracts the profile picture from linkdin

# Where should the reviewer start?
app/routes/applicants/$applicantId/index.tsx
app/routes/projects/$projectId/index.tsx

# How should this be manually tested?
1. Log in to linkdin and fill out the form.
2. Apply to any project
3. login with google
4. Verify that the applicant's photo is displayed in /applicants.
5. Verify that the project you applied to shows the applicant correctly.

# Any background context you want to provide?

# What are the relevant tickets?
#229 

# Screenshots
Captura de pantalla 2023-11-07 a la(s) 8 58 40 a m

Captura de pantalla 2023-11-07 a la(s) 8 58 48 a m

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: app/routes/projects/$projectId/index.tsx [hunk 1]
```diff
@@ -644,6 +605,45 @@ export default function ProjectDetailsPage() {
         handleCloseModal={() => setShowJoinModal(false)}
       />
 
+      <Container sx={{ marginBottom: 2 }}>
```
Comment (martinrobled0) [inline app/routes/projects/$projectId/index.tsx:608]: I think that we can do a new component based on this code in order to use it on other parts of the project

Comment (martinrobled0) [reply app/routes/projects/$projectId/index.tsx:608]: Also if there is no applicants the containers looks like this 
image
without a card component, if you move this to a new component we can improve that

**Review by martinrobled0 — CHANGES_REQUESTED** [review]

**Review by maubarraganwize — APPROVED** [review]
Looks and works ok to me

**Review by beatriz-sanchez-wizeline — APPROVED** [review]
Works for me and the Applicants section has been removed when no applicants have applied.

**Review by martinrobled0 — APPROVED** [review]

**Pull request merged**